features/specs/gezag-meerderjarige/gezag-kind-niet-te-bepalen.feature

```diff
@@ -0,0 +1,24 @@
+#language: nl
+
+Functionaliteit: gezag van een meerderjarige over een kind wanneer gezag niet te bepalen is
+
+
+  Regel: een meerderjarige krijgt voor een minderjarig kind waarvoor het gezag niet bepaald kan worden geen gezagsrelatie geleverd
+
+    Scenario: persoon heeft een minderjarig kind waarover het gezag niet bepaald kan worden
+      Gegeven de persoon 'Ingrid' met burgerservicenummer '000000012'
+      * is ingeschreven in de BRP
+      * is meerderjarig
+      En de persoon 'Jaimy' met burgerservicenummer '000000036'
+      * is ingeschreven in de BRP
+      * is minderjarig
+      * is in Nederland geboren
+      * is geëmigreerd geweest
+      * heeft 'Ingrid' als ouder 1
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
```

Comment:
```suggestion
      | burgerservicenummer | 000000012 |
```

Het is de bedoeling om te laten zien wat er gebeurd wanneer gezag wordt gevraagd over kinderen vanuit het perspectief van de ouder (meerderjarige). Volgens mij hoort hier dan de burgerservicenummer van Ingrid in te staan.